test/features/can_refresh_a_closed_record_test.rb

```diff
@@ -0,0 +1,233 @@
+require 'test_helper'
+Dir[Rails.root.join('test/**/*.rb')].each { |f| require f }
+
+class CanRefreshAClosedRecordTest < Capybara::Rails::TestCase
+  include Helpers::UserHelpers
+  include Helpers::ReviewsHelper
+  include Helpers::HomeHelper
+  include Helpers::CollectionsHelper
+
+  before do
+    OmniAuth.config.test_mode = true
+    mock_login(role: 'admin')
+
+    stub_request(:get, "#{Cmr.get_cmr_base_url}/search/granules.echo10?concept_id=G309210-GHRC")
```

Comment:
This stub is not needed for tests to pass

Reply:
Ok, I removed it from my master